static/css/style.css

```diff
@@ -190,5 +196,19 @@ hr {
 */
 .modal,
 .modal-open {
-    padding-right: 0px !important;
+  padding-right: 0px !important;
+}
+
+/* General movie poster list styles */
+ul#movie_list {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The properties margin-top, margin-bottom, margin-left, margin-right can be replaced by margin.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4672309)